packages/flutter_tools/test/general.shard/fuchsia/fuchsia_kernel_compiler_test.dart

```diff
@@ -0,0 +1,52 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter_tools/src/build_info.dart';
+import 'package:flutter_tools/src/fuchsia/fuchsia_kernel_compiler.dart';
+
+import '../../src/common.dart';
+
+void main() {
+  group('Fuchsia Kernel Compiler', () {
+    final FuchsiaKernelCompiler fuchsiaKernelCompiler = FuchsiaKernelCompiler();
+    test('provide correct flags for release mode', () {
+      expect(
+          fuchsiaKernelCompiler.getBuildInfoFlags(
+            buildInfo: BuildInfo.release,
+            manifestPath: '',
+          ),
+          allOf(<Matcher>[
```

Comment:
I think this can be simplified a bit to:

```contains(<String>['-Ddart.vm.profile=false', '-Ddart.vm.product=true'])```

Reply:
Hmm I think this is going to match `[['-Ddart.vm.profile=true', '-Ddart.vm.product=false'],'--aot']` only

```
Expected: (contains ['-Ddart.vm.profile=true', '-Ddart.vm.product=false'])
  Actual: [
            '--aot',
            '--tfa',
            '--no-embed-sources',
            '-Ddart.vm.profile=true',
            '-Ddart.vm.product=false',
            '-Ddart.developer.causal_async_stacks=false'
          ]
```

Reply:
Ahh, right. Matchers are hard :)